clojure-ts-mode.el

```diff
@@ -821,11 +838,11 @@ Skip the optional metadata node at pos 0 if present."
        n)
      t)))
 
-(defun clojure-ts--node-with-metadata-parent (node)
-  "Return parent for NODE only if NODE has metadata, otherwise return nil."
-  (when-let* ((prev-sibling (treesit-node-prev-sibling node))
-              ((clojure-ts--metadata-node-p prev-sibling)))
-    (treesit-node-parent (treesit-node-parent node))))
+(defun clojure-ts--first-value-child (node)
+  "Return the first value child of the NODE.
```

Comment:
I'd probably add some example to this docstring, as it's not super clear what kind of nodes have "value" to begin with.